.pre-commit-config.yaml

```diff
@@ -55,3 +55,16 @@ repos:
         'python-dateutil', 'pytz', 'six', 'pyarrow', 'chardet', 'fastavro',
         'python-snappy', 'charset-normalizer', 'psutil', 'scipy', 'requests',
         'networkx','typing-extensions']
+  # Pyupgrade - standardize and modernize Python syntax for newer versions of the language
+  - repo: https://github.com/asottile/pyupgrade
+    rev: v3.3.0
+    hooks:
+      - id: pyupgrade
+        args: ["--py37-plus"]
+  # Autoflake - cleanup unused variables and imports
+  - repo: https://github.com/PyCQA/autoflake
+    rev: v2.0.0
+    hooks:
+      - id: autoflake
+        args:
+          - "--in-place"
```

Comment:
Sorry, yes, this is from autoflake - it is in their docs as the last line of their [Introduction](https://github.com/PyCQA/autoflake#introduction:
```
autoflake also removes useless pass statements by default.
```
This can be disabled trivially in the pre-commit config (`--ignore-pass-statements` or `--ignore-pass-after-docstring` as args), let me know if you'd like me to do that.

Reply:
ah there it is -- I think I'd prefer to leave pass in for now. So let's add those flags in the `pre-commit` section for `autoflake`.

Reply:
No problem - `--ignore-pass-statements` or `--ignore-pass-after-docstring` or both?

Reply:
For now, maybe let's just do `--ignore-pass-statements`. Thx

Reply:
I'm good either way. I approved the PR for now even though this is not in yet.

I'll re-review quickly after this change has been made.